src/index.ts

```diff
@@ -7,8 +7,7 @@ const biomePlugin = (options: Options = { mode: 'lint', files: '.', applyFixes:
   const executeCommand = async () => {
     const filesPath = path.join(process.cwd(), options.files ?? ".").replace(/(\\\s+)/g, '\\\\$1');
     const commandBase = `npx @biomejs/biome`;
-    const command = `${commandBase} ${options.mode} "${filesPath}" ${options.applyFixes ? (options.mode === 'format' ? '--write' : '--apply') : ''
-      } --colors=force`;
+    const command = `${commandBase} ${options.mode} "${filesPath}" ${options.applyFixes ? '--write' : ''} ${options.unsafe ? '--unsafe' : ''} --colors=force`;
```

Comment:
```suggestion
    const command = `${commandBase} ${options.mode} "${filesPath}" ${options.applyFixes ? '--write' : ''} ${(options.applyFixes && options.unsafe) ? '--unsafe' : ''} --colors=force`;
```
I don't think one should be able to use `--unsafe` without `applyFixes = true`, right?

Reply:
sure! you are totally right